### DISCUSSION TITLE
Allow non-admin user to sort collections in the sidebar

### DISCUSSION THREAD

**Discussion by beliven-valentino-cossar** [outline/outline#7732] [Ideas]
Hi everyone,

I would like to propose a feature that allows non-admin users to sort collections in the sidebar. Currently, this capability is limited to admin users.

**Benefits**:
- **Inclusive collaboration**: Enables all team members to manage their view, enhancing collaboration.
- **Flexibility**: Users can prioritize collections based on personal, company or project needs.

**Considerations**:
- Implementing a drag-and-drop interface for sorting could be useful.
- Admins could control which roles have this sorting capability.

I’m interested in hearing your thoughts on this proposal.

Thanks!

Valentino

**Comment by tommoor** [comment]
The sidebar sorting is shared across all users, it's not really compatible to have both user sorting _and_ admin level sorting, though I see your other request and grouping is something that's come up quite a few times and makes a lot of sense

**Comment by beliven-valentino-cossar** [comment]
Hi @tommoor, thank you for the quick reply.

Just a little clarification, because I would like to avoid misunderstandings, I didn't mean to suggest having two different sorting options, one for admin users and one for non-admin users. Instead, I would like both admin users and non-admin users (like editors) to be able to modify the sorting of the collections in the sidebar, which is available to all users.
